app/chain_api.py

```diff
@@ -64,3 +64,18 @@ def __init__(self, services):
         if request.method == 'PUT' and index == 'core_operation':
             self.loop.create_task(self.operation_svc.run(output))
         return output
+
+    async def rest_state_control(self, request):
+        body = await request.json()
+        state = body.get('state')
+        valid_states = ('running', 'paused', 'run_one_link')
```

Comment:
valid_states should use the enum/dictionary I mentioned in the core code review.

I tend to prefer a dictionary return block instead of an if/else one on an API (in Python). Can you try converting this into such a block (similar to rest_core in this same class).